scylla/src/utils/pretty.rs

```diff
@@ -66,7 +66,12 @@ where
             )?,
             CqlValue::Counter(c) => write!(f, "{}", c.0)?,
             CqlValue::Date(CqlDate(d)) => {
-                let days_since_epoch = chrono::Duration::days(*d as i64 - (1 << 31));
+                // This is basically a copy of the code used in `impl TryInto<NaiveDate> for CqlDate` impl
+                // in scylla-cql. We can't call this impl because it is behind chrono feature in scylla-cql.
+
+                // date_days is u32 then converted to i64 then we subtract 2^31;
+                // Max value is 2^31, min value is -2^31. Both values can safely fit in chrono::Duration, this call won't panic
```

Comment:
If we anyway use `chrono` in `scylla` unconditionally, why we even put it under feature in `scylla_cql`? What benefit could one have from disabling `chrono` in `scylla_cql` when it's still required in `scylla` ?

Reply:
Maybe less code generated for scylla-cql?

Reply:
This seems to be a very small gain compared to complexity added. Especially that, as you can see, we handle `CqlDate` with `chrono` anyway.